react/tic-tac-toe/src/redux/users/actions.js

```diff
@@ -0,0 +1,28 @@
+import { push } from 'connected-react-router';
+
+import login from '../../services/UsersService';
+import api from '../../config/api';
+
+
+export const actions = {
+  LOGIN: '@@LOGIN/LOGIN',
+  LOGIN_FAILURE: '@@LOGIN/LOGIN_FAILURE',
+  LOGIN_SUCCESS: '@@LOGIN/LOGIN_SUCESS'
+};
+
+const actionsCreators = {
+  login: data => async dispatch => {
+    dispatch({ type: actions.LOGIN });
+    const response = await login(data);
+    if (response.ok) {
+      dispatch({ type: actions.LOGIN_SUCCESS });
+      localStorage.setItem('token', response.data.token);
+      api.setHeader('token', response.data.token);
+      dispatch(push('/game'));
```

Comment:
Use a common constant for your routes addresses

```suggestion
      dispatch(push(ROUTES.GAME));
```